modules/@apostrophecms/image/ui/apos/components/AposMediaManagerEditor.vue

```diff
@@ -71,6 +71,12 @@
           class="apos-media-editor__back" type="outline"
           label="Cancel"
         />
+        <AposButton
```

Comment:
I made a note in the ticket. We should split this into a separate branch to hold until we have proper designs for the image editor. This doesn't seem right with our design patterns.

Reply:
I switched to the trash icon per team discussion.